### PR TITLE
Autom/ac bsa 40 bsa 28

### PR DESCRIPTION
**Pull request by AlanDiCha123**
# Implemented changes
There was duplicated methods and a duplicated file. I had to remove it and implement new logic between Gilberto's methods and mines

image
The last successful test is of my method

### CODE REVIEW

**Review by LanderoX144 — APPROVED** [review]

File: Pages/fund_transfer.py
Comment (eduxsan) [inline Pages/fund_transfer.py:1]: PLease change file name to `fund_transfer_page.py`

File: Tests/Fund_Transfer/test_fund_transfer.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        fund_transfer.write_payers_account("131294")
+        fund_transfer.write_payees_account("131295")
+        fund_transfer.write_amount("500")
+        fund_transfer.write_description("nonas")
+        fund_transfer.click_on_submit()
+        time.sleep(5)
+        fund_transfer.verify_data_by_field("From", "131294")
+        fund_transfer.verify_data_by_field("To", "131295")
+        fund_transfer.verify_data_by_field("Amount", "500")
+        fund_transfer.verify_data_by_field("Description", "nonas")
```
Comment (eduxsan) [inline Tests/Fund_Transfer/test_fund_transfer.py]: Please revert the transfer, to avoid the origin account being empty in the future

**Review by eduxsan — CHANGES_REQUESTED** [review]
Good job! Please take a look at my comments

**Review by eduxsan — APPROVED** [review]

File: Tests/New_Customer/test_new_customer.py [hunk 2]
```diff
@@ -92,7 +91,7 @@ def test_BSA_1_GLA(self):
 
         addCustomer = NewCustomerPage(self.driver)
         time.sleep(2)
-        addCustomer.write_on_input("Customer Name","Holmes")
+        addCustomer.write_customer_name("Holmes")
```
Comment (LanderoX144) [inline Tests/New_Customer/test_new_customer.py]: why did you chance this metod? :0

**Review by LanderoX144 — CHANGES_REQUESTED** [review]

File: Tests/New_Customer/test_new_customer.py (hunk 2 above)
Comment (AlanDiCha123) [reply Tests/New_Customer/test_new_customer.py]: I've returned your method back

**Review by LanderoX144 — APPROVED** [review]
:3 BONO POR EFICIENCIA

**Pull request merged**